tests/sqllogictests/suites/base/05_ddl/05_0026_ddl_unset_settings.test

```diff
@@ -58,3 +50,34 @@ show settings like 'DATA_RETENTION_TIME_IN_DAYS';
 ----
 data_retention_time_in_days 1 1 [0, 90] DEFAULT Sets the data retention time in days. UInt64
 
+statement ok
+set global load_file_metadata_expire_hours=12;
+
+query TT
+show settings like 'load_file_metadata_expire_hours';
+----
+load_file_metadata_expire_hours 12 168 [0, 18446744073709551615] GLOBAL Sets the hours that the metadata of files you load data from with COPY INTO will expire in. UInt64
+
+statement ok
+set load_file_metadata_expire_hours=13;
+
+query TTTTTTT
+show settings like 'load_file_metadata_expire_hours';
+----
+load_file_metadata_expire_hours 13 168 [0, 18446744073709551615] SESSION Sets the hours that the metadata of files you load data from with COPY INTO will expire in. UInt64
+
+statement ok
+unset session load_file_metadata_expire_hours;
+
+query TTTTTTT
+show settings like 'load_file_metadata_expire_hours';
+----
+load_file_metadata_expire_hours 12 168 [0, 18446744073709551615] GLOBAL Sets the hours that the metadata of files you load data from with COPY INTO will expire in. UInt64
+
+statement ok
+unset load_file_metadata_expire_hours
+
+query TTTTTTT
+show settings like 'load_file_metadata_expire_hours';
```

Comment:
`load_file_metadata_expire_hours` default value is changed:  https://github.com/datafuselabs/databend/pull/15514

Reply:
Fixed